froide/comments/templates/froide_comments/form.html

```diff
@@ -22,10 +22,26 @@
 
   {% render_field form.comment %}
 
+  {% is_moderator as is_moderator %}
```

Comment:
The variable name should not shadow the template tag name to avoid confusion.

Reply:
sure